src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -276,6 +307,32 @@ export class CourseOverviewComponent implements OnInit, OnDestroy, AfterViewInit
         }
     }
 
+    /** initialize courses attribute by retreiving recently accessed courses from the server */
+    getRecentlyAccessedCourses() {
```

Comment:
For me, `get` is not a good keyword in the method name, was I would expect that method to not have any side-effects on any attributes.

Maybe `updateRecentlyAccessedCourses` would be a better name?

Reply:
done 👍